### PR TITLE
Bugfix/start up

### PR DESCRIPTION
**Pull request by bernardolm**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -130,4 +130,8 @@ dist
 .pnp.*
 
 # next build file
-build/
\ No newline at end of file
+build/
+
+*.env*
+*credentials*
+yarn.lock
```
Comment (brunofurmon) [inline .gitignore]: yarn.lock should be versioned to prevent version "unlocking" and sudden build breaks.
```suggestion
```

File: Dockerfile [hunk 2]
```diff
@@ -1,13 +1,13 @@
 FROM node:20.9-slim AS base
-ENV APPDIR /usr/open-org-chart
+ENV APPDIR=/usr/open-org-chart
```
Comment (brunofurmon) [inline Dockerfile:2]: I received a log warning from docker that `ENV KEY=VALUE` is legacy and that we should use `ENV KEY VALUE`

Comment (bernardolm) [reply Dockerfile:2]: I received the exact opposite warning

{A6B69A1E-237A-445F-BD76-708F12E8C9D8}

File: .gitignore (hunk 1 above)
Comment (bernardolm) [reply .gitignore]: Fixed

File: Dockerfile (hunk 2 above)
Comment (brunofurmon) [reply Dockerfile:2]: OH! that's it then :) Thanks!

Comment (brunofurmon) [reply Dockerfile:2]: I misinterpreted the message, you're correct!

**Review by brunofurmon — APPROVED** [review]

**Pull request merged**